testapi.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+import requests
+import json
+import re
+from app import app
+
+
+class TestMailApi(unittest.TestCase):
+    def test_sendmail(self):
+        response = requests.post('http://localhost:5000/api/mail/send',
+                                 headers={'Client-Key': app.config["CLIENT_KEY"], 'Content-Type': 'application/json'},
+                                 json={"to": app.config["SMTP_USER"],
+                                       "from": app.config["SMTP_USER"],
+                                       "subject": "test mail",
+                                       "text": "test"})
+        self.assertTrue(response.text.find('"status_code": 200'))
```

Comment:
I'm wondering what would be the code coverage of the unit-tests.

Reply:
i've added two test cases for both api calls